src/Ninja/G1CP/Content/Tests/test228.d

```diff
@@ -0,0 +1,44 @@
+/*
+ * #228 Gorn can't be asked about after talking to Lares
+ *
+ * There does not seem an easy way to test this fix programmatically, so this test relies on manual confirmation.
+ *
+ * Expected behavior: Gorn is findable after talking to Lares.
```

Comment:
A more descriptive expected behavior is that the AI variable is correctly updated. (Whether the NPC is "findable" is not checked in the text.)